src/frontends/src/query_frontend_main.cpp

```diff
@@ -11,18 +11,21 @@ int main(int argc, char* argv[]) {
   // clang-format off
   options.add_options()
     ("redis_ip", "Redis address",
-        cxxopts::value<std::string>()->default_value("localhost"))
+        cxxopts::value<std::string>()->default_value(clipper::DEFAULT_REDIS_ADDRESS))
     ("redis_port", "Redis port",
-        cxxopts::value<int>()->default_value("6379"))
+        cxxopts::value<int>()->default_value(std::to_string(clipper::DEFAULT_REDIS_PORT)))
     ("threadpool_size", "Number of threads for the task execution threadpool",
```

Comment:
You can actually delete this command line arg and corresponding config field entirely. It's no longer used. Instead we create a thread in the task execution threadpool for each connected container.